epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -287,11 +287,11 @@ public void onViewDetachedFromWindow(T view) {
 
   @Override
   public String toString() {
-    return getClass().getSimpleName() + "{" +
-        "id=" + id +
-        ", layout=" + getLayout() +
-        ", shown=" + shown +
-        ", addedToAdapter=" + addedToAdapter +
-        '}';
+    return getClass().getSimpleName() + "{"
+        + "id=" + id
```

Comment:
Yes, this is the formatting I got from Felipe. Although I think I may have changed a few things to my preference. In general we should follow the google java style guide.